Use CONFIG_TARGET_ROOTFS_PARTSIZE only when it is defined

diff --git a/include/image.mk b/include/image.mk
--- a/include/image.mk
+++ b/include/image.mk
@@ -231,7 +231,7 @@ define Image/mkfs/targz
 	$(TAR) -czpf $(BIN_DIR)/$(IMG_PREFIX)$(if $(PROFILE),-$(PROFILE))-rootfs.tar.gz --numeric-owner --owner=0 --group=0 -C $(TARGET_DIR)/ .
 endef
 
-E2SIZE=$(shell echo $$(($(CONFIG_TARGET_ROOTFS_PARTSIZE)*1024*1024)))
+E2SIZE=$(shell echo $$(($(if $(CONFIG_TARGET_ROOTFS_PARTSIZE),$(CONFIG_TARGET_ROOTFS_PARTSIZE),1)*1024*1024)))
 
 define Image/mkfs/ext4
 	$(STAGING_DIR_HOST)/bin/make_ext4fs \
